src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -1180,10 +1180,10 @@ namespace winrt::TerminalApp::implementation
             // we should save our state.
             if (_root->ShouldUsePersistedLayout(_settings) && _numOpenWindows == 1)
             {
-                if (auto layout = _root->GetWindowLayout())
+                if (const auto layout = _root->GetWindowLayout())

```

Comment:
yeah, do blame com (or, winrt) for this. const isn't in its object model, so i guess that what we are modelling is that ... uh, the pointer inside the `layout` smart object won't change? _yay_